.github/workflows/ci.yml

```diff
@@ -280,6 +373,8 @@ jobs:
         shard_index: [0, 1]
         include: ${{ fromJson(needs.setup.outputs.test_matrix) }}
     steps:
+      - uses: actions/checkout@v4
```

Comment:
Ditto; the gtest runner does not require anything other than the test binary.

Reply:
Indeed, it's not necessary. Good catch. The fix will be included in the next commit

Reply:
It is added again due to the composite action